PayPalDataCollector/src/main/java/com/braintreepayments/api/PayPalDataCollector.java

```diff
@@ -100,21 +134,50 @@ public void collectDeviceData(@NonNull final Context context, @NonNull final Pay
      * @param riskCorrelationId Optional client metadata id
      * @param callback          {@link PayPalDataCollectorCallback}
      */
-    public void collectDeviceData(@NonNull final Context context, @Nullable final String riskCorrelationId, @NonNull final PayPalDataCollectorCallback callback) {
+    public void collectDeviceData(
+        @NonNull final Context context,
+        @Nullable final String riskCorrelationId,
+        @NonNull final PayPalDataCollectorCallback callback
+    ) {
+        PayPalDataCollectorRequest request = new PayPalDataCollectorRequest(false);
+        collectDeviceData(context, request, riskCorrelationId, callback);
+    }
+
+    /**
+     * Collects device data for PayPal APIs.
+     * <p>
+     * We recommend that you call this method as early as possible, e.g. at app launch. If that's too early,
+     * call it at the beginning of customer checkout.
+     * <p>
+     * Use the return value on your server, e.g. with `Transaction.sale`.
+     *
+     * @param context                    Android Context
+     * @param payPalDataCollectorRequest params for the data collection request
+     * @param riskCorrelationId          Optional client metadata id
+     * @param callback                   {@link PayPalDataCollectorCallback}
+     */
+    public void collectDeviceData(
+        @NonNull final Context context,
+        @NonNull final PayPalDataCollectorRequest payPalDataCollectorRequest,
+        @Nullable final String riskCorrelationId,
```

Comment:
Ok sorry should have given all this feedback on the initial feature PR, but now thinking for the new non-deprecated method we should also put `riskCorrelationId` on the `PayPalDataCollectorRequest` and keep all request params in that object

Reply:
Then we can also eliminate this overload method I think and just have one method with the request object

Reply:
No worries at all. The initial PR review was rushed since we were trying to get the snapshot out the door.

Great suggestion - let me add the `riskCorrelationId` to the request object.